src/pudl/glue/ferc1_eia.py

```diff
@@ -247,8 +252,21 @@ def get_raw_plants_ferc1(
         )
         plant_dfs.append(plant_df)
 
+    all_plants = pd.concat(plant_dfs)
+    most_recent_year = max(all_plants.report_year)
```

Comment:
This is a single integer representing the most recent year of data that's available for any plant in the table of all plants, right? Is it appropriate to select only plant attributes from that year for mapping purposes? It'll leave out information about plants that don't show up in the most recent report year, won't it? Is that okay?

Reply:
yes that is what is happening here. I think it is probably not ideal in weird edge cases but not the end of the world. this is for grabbing a little bit of information about the plants in order to display to the manual mappers to help with determining whether or not a plant needs to be linked to EIA. In a world in which we have mapped all of the old ferc plant data we are ever going to get, most unmapped plants will be integrated will be plants from recent years/

I could certainly imagine employing a selection that grabbed the most recent year of "complete" data for each plant, but that would introduce additional complexities into a thing that is really just an aid to determine how important a plant is/how much attention to pay it.